cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2a.prm

```diff
@@ -0,0 +1,111 @@
+# This cookbook models oceanic subduction in a 2d
+# box of 3000 x 670 km.
+# The subducting plate is driven by inflow boundary
+# conditions on the top part of the right boundary,
+# while a balancing outflow is prescribed underneath.
+# Only density and viscosity differ per composition.
+# Building on case 1, an initial temperature field
+# based on the plate cooling model and
+# a mantle adiabatic gradient of 0.25 degrees/km
+# is prescribed, but temperature still does not affect
+# the material parameters.
+# High resolution is required to a.o. resolve the weak zone
+# between the subducting and overriding plate, so about
+# 100 MPI processes are required.
```

Comment:
Can you provide a range of MPI processes here? Even better would be to say something like `.. overriding plate, which results in a model size of approximately X degrees of freedom. This requires on the order of X-Y processors to run efficiently over X-Y hours.`